src/containers/BooksForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Select from 'react-select';
+
+function BooksForm() {
```

Comment:
### Optional
- [ ] It would be better to use ES6 arrow syntax since we are using most of the ES6 features, maintaining consistency is our codebase is a good practice